drivers/mfd/intel-m10-bmc-pmci.c

```diff
@@ -1,148 +1,242 @@
 // SPDX-License-Identifier: GPL-2.0
 /*
- * PMCI-based interface to MAX10 BMC
- *
- * Copyright (C) 2020-2021 Intel Corporation, Inc.
+ * MAX10 BMC Platform Management Component Interface (PMCI) based
+ * interface.
  *
+ * Copyright (C) 2020-2023 Intel Corporation.
  */
 
 #include <linux/bitfield.h>
+#include <linux/device.h>
 #include <linux/dfl.h>
+#include <linux/mfd/core.h>
 #include <linux/mfd/intel-m10-bmc.h>
+#include <linux/minmax.h>
```

Comment:
CentOS 8.4 does not have this include file:
  CC [M]  drivers/mfd/intel-m10-bmc-pmci.o
drivers/mfd/intel-m10-bmc-pmci.c:14:10: fatal error: linux/minmax.h: No such file or directory
 #include <linux/minmax.h>
          ^~~~~~~~~~~~~~~~
compilation terminated.